admin/controllers/check-form.js

```diff
@@ -68,15 +68,15 @@ const uploadAndViewForms = async (req, res, next) => {
 const removeCheckForm = async (req, res, next) => {
   const id = req.params.formId
   try {
-    const CheckWindow = await checkFormDataService.getActiveForm(id)
-    if (!CheckWindow) {
+    const checkForm = await checkFormDataService.getActiveForm(id)
+    if (!checkForm) {
       return next(new Error(`Unable to find form.id [${id}]`))
```

Comment:
could you please change this to `Unable to find check form with id [${id}]`

i appreciate you may not have written it, but it will aid us with potential debugging

Reply:
Done, it will be in the next push.